jetstream/workflows/run.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Workflow
+metadata:
+  generateName: jetstream-
+spec:
+  entrypoint: jetstream
+  onExit: export-statistics # export statistics after all analyses
+  arguments:
+    parameters:
+    - name: experiments  # set dynamically when worklfow gets deployed
+  templates:
+  - name: jetstream
+    parallelism: 5  # run up to 5 containers in parallel at the same time
+    inputs:
+      parameters:
+        - name: experiments
+    steps:
+    - - name: analyse-experiment      
+        template: analyse-experiment  
+        arguments:
+          parameters:
+          - name: slug
+            value: "{{item.slug}}"
+          - name: date
+            value: "{{item.date}}"
+        withParam: "{{inputs.parameters.experiments}}"  # process these experiments in parallel
+
+  - name: analyse-experiment
+    inputs:
+      parameters:
+      - name: date
+      - name: slug
+    container:
+      image: gcr.io/moz-fx-data-experiments/jetstream:latest
+      command: [bin/entrypoint, run, "--date={{inputs.parameters.date}}", "--experiment_slug={{inputs.parameters.slug}}"]
```

Comment:
in an earlier comment i think i'm suggesting that we might want to be able to provide a project_id and a dataset_id here